medicines/web/src/services/events.ts

```diff
@@ -0,0 +1,26 @@
+import TagManager from 'react-gtm-module';
+
+const pushToDataLayer = (dataLayer: any) => {
+  TagManager.dataLayer({
+    dataLayer,
+  });
+};
+
+export default {
+  search: (searchTerm: string, pageNo: number) => {
+    pushToDataLayer({
+      event: 'search',
+      searchTerm,
+      pageNo,
+    });
+  },
+  substances: (substance: string) => {
+    pushToDataLayer({ event: 'substance', substance });
```

Comment:
Mixed pluralisation here. Should `substances` and `substance` be the same word?

Reply:
I can rename this method to `viewProductsForSubstance` if it makes things clearer. I can do similar renames for the other actions, like `viewSubstancesStartingWith` below.

Reply:
Done